src/main/java/org/ssor/boss/core/exception/UserDataAccessException.java

```diff
@@ -0,0 +1,12 @@
+package org.ssor.boss.core.exception;
+
+public class UserDataAccessException extends RuntimeException {
+
+	/**
+	 * @param message
+	 */
+	public UserDataAccessException(String message) {

```

Comment:
@christianangeles The brace here as well
```suggestion
	public UserDataAccessException(String message) 
	{

```